### PR TITLE
Vermont taxable income standard deductions

### LINKED ISSUE #2601: Vermont taxable income standard deductions
image

Legal Code: PlaceHolder
Tax Form: https://tax.vermont.gov/sites/tax/files/documents/IN-111-2022.pdf  Taxable Income Line 4
Instruction: https://tax.vermont.gov/sites/tax/files/documents/Income%20Booklet-2022.pdf Taxable Income Line 4

### PR DESCRIPTION
**Pull request by mzx-4real**
Fixes #2601

### Summary
📝📊🧮

This pull request adds Vermont standard deductions to the policy engine model, using new parameters, variables, and tests. It also updates the changelog entry to reflect this minor feature addition.

> _`vt_deductions`_
> _New feature for the model_
> _Autumn tax reform_

### Walkthrough
*  Add Vermont standard deductions to the model ([link](https://github.com/PolicyEngine/policyengine-us/pull/2602/files?diff=unified&w=0#diff-5885042762833a4aef51405a4036067e4e6aefa7a5091a2987d1d9d1fc2647e8R1-R4))
   * Create new parameter files for the base and extra standard deduction amounts for each filing status ([link](https://github.com/PolicyEngine/policyengine-us/pull/2602/files?diff=unified&w=0#diff-aaf73cb398262a3ad3698b79f190505356d775d32b72d96d08b1c98f3d1fb1b0R1-R32), [link](https://github.com/PolicyEngine/policyengine-us/pull/2602/files?diff=unified&w=0#diff-b996fbd5329d1490c4292c9532a77930083f813c33474b5eb8c644f6dae08c09R1-R36))
   * Create new variable file for `vt_standard_deductions`, which calculates the deduction based on the filing status, the aged or blind count, and the parameters ([link](https://github.com/PolicyEngine/policyengine-us/pull/2602/files?diff=unified&w=0#diff-3be640804caa754a4b65cfe0369948462982863bd51d9ea201043c86d7805454R1-R25))
   * Modify the `vt_deductions` variable to include `vt_standard_deductions` in the list of deductions ([link](https://github.com/PolicyEngine/policyengine-us/pull/2602/files?diff=unified&w=0#diff-5b4da40c9563285f3becaa835d116595723e546203c2b09ecc10d719bcb77828L10-R13))
   * Create new test file for `vt_standard_deductions`, with several test cases for different scenarios and expected outputs ([link](https://github.com/PolicyEngine/policyengine-us/pull/2602/files?diff=unified&w=0#diff-72247bcd5cbe6660443894ee0e07e926d83871e5ca5fc6b4d9b89bea7e77fe4dR1-R71))

### CODE REVIEW

File: policyengine_us/parameters/gov/states/vt/tax/income/deductions/standard/base.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+description: Vermont provides a base standard deduction of this amount.
+metadata:
+  label: VT base standard deduction
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/deductions/standard/base.yaml:3]:
```suggestion
  label: Vermont base standard deduction
```

File: policyengine_us/parameters/gov/states/vt/tax/income/deductions/standard/extra.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
+description: Vermont provides this extra standard deduction amount for each aged/blind head/spouse.
+metadata:
+  label: Vermont extra standard deduction amount for each aged/blind head/spouse
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/deductions/standard/extra.yaml:3]:
```suggestion
  label: Vermont additional aged or blind, head or spouse, standard deduction amount
```

File: policyengine_us/parameters/gov/states/vt/tax/income/deductions/standard/extra.yaml
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/deductions/standard/extra.yaml:1]: we can make this a single amount parameter as the values stay the same no matter the filing status

File: policyengine_us/variables/gov/states/vt/tax/income/taxable_income/vt_standard_deductions.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+
+class vt_standard_deductions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "VT standard deduction"
+    unit = USD
+    definition_period = YEAR
+    documentation = (
+        "Vermont standard deduction, including bonus for aged or blind."
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/vt/tax/income/taxable_income/vt_standard_deductions.py:11]:
```suggestion
        "Vermont standard deduction."
```

File: policyengine_us/variables/gov/states/vt/tax/income/taxable_income/vt_exemptions.py
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/vt/tax/income/taxable_income/vt_exemptions.py:1]: did you mean to delete this file?

File: policyengine_us/variables/gov/states/vt/tax/income/taxable_income/vt_deductions.py
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/vt/tax/income/taxable_income/vt_deductions.py:1]: we can use this file once we have the itemized deductions

File: policyengine_us/parameters/gov/states/vt/tax/income/deductions/standard/base.yaml [hunk 4]
```diff
@@ -0,0 +1,30 @@
+description: Vermont provides a standard deduction of this amount.
```
Comment (PavelMakarchuk) [inline policyengine_us/parameters/gov/states/vt/tax/income/deductions/standard/base.yaml:1]:
```suggestion
description: Vermont provides this standard deduction based on filing status.
```

File: policyengine_us/tests/policy/baseline/gov/states/vt/vt_standard_deductions.yaml [hunk 5]
```diff
@@ -0,0 +1,71 @@
+- name: 2021, Single Filer, with zero additional standard deduction
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/states/vt/vt_standard_deductions.yaml:1]:
```suggestion
- name: 2021, Single filer, with zero additional standard deduction
```

File: policyengine_us/tests/policy/baseline/gov/states/vt/vt_standard_deductions.yaml [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+  period: 2021
+  input:
+    filing_status: SINGLE
+    aged_blind_count: 0
+    state_code: VT
+  output:
+    vt_standard_deductions: 6_350 #6350+0
+
+- name: 2022, Single Filer, with two additional standard deduction
```
Comment (PavelMakarchuk) [inline policyengine_us/tests/policy/baseline/gov/states/vt/vt_standard_deductions.yaml:10]:
```suggestion
- name: 2022, Single filer, with two additional standard deduction
```

File: policyengine_us/tests/policy/baseline/gov/states/vt/vt_standard_deductions.yaml [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+  period: 2022
+  input:
+    filing_status: SINGLE
+    aged_blind_count: 2
+    state_code: VT
+  output:
+    vt_standard_deductions: 8_600 #6500+2*1050
+
+- name: 2021, Joint Filer, with three additional standard deduction
```
[the remark this answers is not in the document]
Comment (PavelMakarchuk) [reply policyengine_us/tests/policy/baseline/gov/states/vt/vt_standard_deductions.yaml:19]: all

File: policyengine_us/variables/gov/states/vt/tax/income/taxable_income/vt_standard_deductions.py [hunk 8]
```diff
@@ -0,0 +1,26 @@
+from policyengine_us.model_api import *
+
+
+class vt_standard_deductions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "VT standard deduction"
```
Comment (PavelMakarchuk) [inline policyengine_us/variables/gov/states/vt/tax/income/taxable_income/vt_standard_deductions.py:7]:
```suggestion
    label = "Vermont standard deduction"
```

**Review by PavelMakarchuk — APPROVED** [review]

File: policyengine_us/parameters/gov/states/vt/tax/income/deductions/standard/additional_amount.yaml [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+metadata:
+  label: Vermont additional aged or blind, head or spouse, standard deduction amount
+  period: year
+  unit: currency-USD
+  reference:
+    - title: 2021 VERMONT Income Tax Return Form Instructions-Taxable Income Line 4
+      href: https://tax.vermont.gov/sites/tax/files/documents/Income-Booklet-2021.pdf
+    - title: 2022 VERMONT Income Tax Return Form Instructions-Taxable Income Line 4
+      href: https://tax.vermont.gov/sites/tax/files/documents/Income%20Booklet-2022.pdf 
+    - title: Legal Code Titl. 32 V.S.A. § 5811(21)(C)(iii) # Legal code specification amount date unknown.
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/vt/tax/income/deductions/standard/additional_amount.yaml:11]:
```suggestion
     # Legal code specifies $1,000 but no effective date.
    - title: Legal Code Titl. 32 V.S.A. § 5811(21)(C)(iii)
```

File: policyengine_us/variables/gov/states/vt/tax/income/deductions/vt_standard_deduction.py [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+        "http://legislature.vermont.gov/statutes/section/32/151/05811",  # Titl. 32 V.S.A. § 5811(21)(C)(ii)(iii)
+    )
+    defined_for = StateCode.VT
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.vt.tax.income.deductions.standard
+        # Base deduction based on filing status.
+        filing_status = tax_unit("filing_status", period)
+        base_deduction = p.base[filing_status]
+        # Aged/blind additional standard deduction.
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/vt/tax/income/deductions/vt_standard_deduction.py:21]:
```suggestion
        # Vermont mirrors the federal definition of aged/blind by citing 26 U.S.C. § 63(f).
        # The aged_blind_count variable captures this, for head and spouse.
```

File: policyengine_us/parameters/gov/states/vt/tax/income/deductions/standard/additional_amount.yaml
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/vt/tax/income/deductions/standard/additional_amount.yaml:1]: rename `additional.yaml`

File: policyengine_us/variables/gov/states/vt/tax/income/deductions/vt_standard_deduction.py [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+        aged_blind_count = tax_unit("aged_blind_count", period)
+        amount_per_aged_blind = p.additional_amount
+        aged_blind_deduction = aged_blind_count * amount_per_aged_blind
```
Comment (MaxGhenis) [inline policyengine_us/variables/gov/states/vt/tax/income/deductions/vt_standard_deduction.py:24]:
```suggestion
        aged_blind_deduction = aged_blind_count * p.additional
```

**Review by MaxGhenis — CHANGES_REQUESTED** [review]

**Review by PavelMakarchuk — APPROVED** [review]

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**